extra_tests/regression_checks.py

```diff
@@ -0,0 +1,45 @@
+"""
+    Regression tests
+"""
+
+from n3fit.tests.test_fit import check_fit_results, EXE
+import pathlib
+from reportengine.compat import yaml
+import shutil
+import subprocess as sp
+
+import pytest
+
+
+
+REGRESSION_FOLDER = pathlib.Path(__file__).with_name("regression_fits")
+
+# Avoid always round-number replicas or 1/2
+runcard_and_replicas = {
+        "normal_fit": 72,
+        "central": 16,
+        "diagonal": 45,
+        "feature_scaling": 81,
+        "flavour": 29,
+        "no_msr": 92,
+        "no_sumrules": 18,
+        "no_vsr": 54,
+        "trainable_prepro": 61,
+        "no_lagrange": 27
+        }
+
+@pytest.mark.parametrize("runcard_replica", runcard_and_replicas.items())
+def test_regression_fit(tmp_path, runcard_replica):
+    runcard, replica = runcard_replica
```

Comment:
```suggestion
@pytest.mark.parametrize("runcard,replica", runcard_and_replicas.items())
def test_regression_fit(tmp_path, runcard, replica):
```

This fixes the test ids to be more informative.

Reply:
didn't realise you could do that with parametrize! Thanks!